Library/Formula/corectl.rb

```diff
@@ -0,0 +1,42 @@
+require "language/go"
+
+class Corectl < Formula
+  desc "CoreOS over OS X made very simple"
+  homepage "https://github.com/TheNewNormal/corectl"
+  url "https://github.com/TheNewNormal/corectl/archive/v0.0.19.tar.gz"
+  sha256 "323ebe7e3f612a2656e4b844793f62433a46604ee78455446e04db6b8bc0247e"
+  head "https://github.com/TheNewNormal/corectl.git", :branch => "golang"
+
+  depends_on "go" => :build
+  depends_on "godep" => :build
+  depends_on "gnu-sed" => :build
+  depends_on :macos => :yosemite
+
+  def install
+    ENV["GOPATH"] = buildpath
+
+    mkdir_p buildpath/"src/github.com/TheNewNormal/"
+    ln_s buildpath, buildpath/"src/github.com/TheNewNormal/#{name}"
+    Language::Go.stage_deps resources, buildpath/"src"
+
+    # args = []
+    # args << "VERSION=#{version}" if build.stable?
+    # system "make", "corectl", *args
+    # busts with "cannot load DWARF output from ""...
+    # similar to https://github.com/jwaldrip/homebrew-utils/issues/1
+    # workaround ...
+    cmd = `make homebrewHack VERSION=#{version}` if build.stable?
+    cmd = `make homebrewHack` if build.head?
```

Comment:
@mikemcquaid - well, i'd say that _a priori_ stripping is to be done after all is glued and not before (i.e. stripping a library (dependency) before it gets linked is unnatural).

FWIW and _at sight_ there's some (homebrew env) variable that's passed to `cgo` (when invoked by `godep`, at least) by `make` that's not passed to it when `godep` is called directly by `system`, and that's what seems to be breaking builds (as the dependencies that are built via `cgo` get stripped prematurely).

the workaround people seems to be using is either just not using `make` or (as i did in latest iteration) assure that those (go)deps get properly _installed_ before any attempt to consume them.

thanks!